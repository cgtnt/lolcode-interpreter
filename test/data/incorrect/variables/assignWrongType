HAI 

I HAS A VARONE ITZ 10
I HAS A VARTWO ITZ A YARN
VARTWO R VARONE

VISIBLE VARTWO 

KTHXBYE
