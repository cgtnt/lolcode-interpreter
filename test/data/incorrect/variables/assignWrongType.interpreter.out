Line 5: Cannot assign 10(NUMBR) to VARTWO(YARN)
